eslint.config.mjs

```diff
@@ -0,0 +1,36 @@
+import bpmnIoPlugin from 'eslint-plugin-bpmn-io';
+
+const files = {
+  test: [ '**/test/**/*.js' ],
+  lib: [ 'lib/**/*.js' ]
+};
+
+export default [
+  {
+    ignores: [
+      'dist',
+      'tmp'
+    ]
+  },
+  ...bpmnIoPlugin.configs.browser.map(config => {
+    return {
+      ...config,
+      files: files.lib
+    };
+  }),
+  ...bpmnIoPlugin.configs.node.map(config => {
+    return {
+      ...config,
+      ignores: files.lib
```

Comment:
This makes test files apply node config.

We need build files, and other files.

Reply:
Yeah that was the intention.

Reply:
We only have browser stuff in certain places.